google-http-client/src/main/java/com/google/api/client/json/webtoken/JsonWebSignature.java

```diff
@@ -574,4 +574,35 @@ public static String signUsingRsaSha256(
             SecurityUtils.getSha256WithRsaSignatureAlgorithm(), privateKey, contentBytes);
     return content + "." + Base64.encodeBase64URLSafeString(signature);
   }
+
+  static class DerEncoder {
+    private static byte DER_TAG_SIGNATURE_OBJECT = 0x30;
+    private static byte DER_TAG_ASN1_INTEGER = 0x02;
+
+    static byte[] encode(byte[] signature) {
```

Comment:
why the fully qualified name?

Reply:
Oops, this was migrated from WIP on the `google-auth-library` which used guava's `Preconditions`. `google-http-client` has legacy wrappers for guava's `Preconditions` (with the same name in the current package) from when it was shaded and included in this jar.

I'll switch to the wrapper to maintain consistency with the rest of the library's usage of Preconditions.